### ISSUE TITLE
Candidate for zim wiki android app: Markor

### ISSUE THREAD

**Issue by ghost** [jaap-karssenberg/zim-android-mockapp#6]
I discovered an actively developed android app called markor: https://github.com/gsantner/markor
its already designed to be used with different file formats (todo.txt and markdown) and it already has a folder based notebook structure. Even in its current state it can be used as a zim wiki android app. The only thing missing is the syntax formatting and maybe todo support. Ive added an issue to implement syntax formatting here https://github.com/gsantner/markor/issues/393

**Comment by starstuff** [comment]
we fork for zim wiki app?

**Comment by pfalcon** [comment]
> we fork for zim wiki app?

No, we'll make Zim support Markdown (the de-facto standard for plaintext-based markup) natively (https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/26), and then will just use Markor or any other Markdown tools (which are many) with Zim content. Because you can't fork (and maintain) everything.

**Comment by fredericjacob** [comment]
Good news for those who are also desperately missing a mobile client for Zim: Markor is going to get support for Zim-Wiki files! I'm currently working on it (together with @pschwede) on gsantner/markor#1098.

Most features (editor highlighting, most text actions, some view mode) are implemented now, so feel free to try it out and give us feedback.
I'm already using the current build productively with my main zim notebook and Syncthing to keep it in sync with the desktop. So far, it works quite well to use the extended Markor app as a mobile companion. :)

**Comment by fredericjacob** [comment]
It's done - Markor 2.5 has just been released on Google Play and F-Droid and contains the new Zim Wiki features!
So finally we have a quite full-fledged unofficial Zim client on Android. :tada:
I'd like to encourage all Zim users to try it out, report potential improvements, suggest new features or even contribute in the future development of these. :)
More information about the new release and the Zim Wiki features can be found on the project website: https://gsantner.net/blog/2020/12/20/markor-v2.5-zimwiki-searchreplace-githubactions-zettelkasten.html?source=github_discussion

**Comment by fredericjacob** [comment]
In the next update, Markor will get additional Zim-related features - including a table of contents for the editor and the ability to insert/attach images or files.

@jaap-karssenberg Would you be interested to promote/mention Markor as an inofficial Android client for Zim on the Zim Desktop website? I really think both projects could benefit from each other, reaching a bigger audience and attracting more users if it is clear that Zim can be used cross-platform. There's a similar scenario with the flashcard program Anki - the official desktop and iOS applications are complemented by an unofficial Android client, which is still mentioned on the website. Works really well as far as I can tell. :)

**Comment by jaap-karssenberg** [comment]
Yes, I will update website to mention Markor - using it myself as well -
however at the moment I do not sync between phone and laptop, so not
sharing data between Markor and Zim myself.

Long term plan is also for Zim to support Markdown. But don't hold your
breath ...
